all-imports.html

```diff
@@ -1 +1 @@
-<link rel="import" href="mtz-ajax-interceptor-behavior.html">
+<link rel="import" href="mtz-status-code-interceptor.html">
```

Comment:
Don't see the point in your all-imports file if it doesn't have *all* imports....